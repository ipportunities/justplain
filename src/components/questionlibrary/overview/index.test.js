import React from 'react'
import { shallow } from 'enzyme'
import QuestionLibraryOverview from './index'

jest.mock('react-redux', () => ({
  ...jest.requireActual('react-redux'),
  useSelector: jest.fn().mockReturnValue({ auth: { token: 'a' } })
}))

describe('Question Library Overview', () => {
  it('should render correctly', () => {
    const component = shallow(<QuestionLibraryOverview />)
    expect(component).toMatchSnapshot()
  })
})
