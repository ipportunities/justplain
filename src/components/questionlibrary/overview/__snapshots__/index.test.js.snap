// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Question Library Overview should render correctly 1`] = `
<div
  className="container-fluid"
>
  <div
    className="row"
  >
    <div
      className="col-6 offset-3"
    >
      <div
        className="row"
      >
        <h1>
          Question Library
        </h1>
      </div>
      <div
        className="row pt-5"
      >
        <table
          className="table table-striped table-bordered"
        >
          <thead
            className="thead"
          >
            <tr>
              <th>
                Question category
              </th>
              <th>
                Amount of questions
              </th>
              <th />
            </tr>
          </thead>
          <tbody>
            <tr>
              <td
                colSpan={99}
              >
                <input
                  className="form-control"
                  id="newCategory"
                  onChange={[Function]}
                  onKeyPress={[Function]}
                  placeholder="Type om een nieuwe categorie toe te voegen"
                  style={
                    Object {
                      "backgroundColor": "transparent",
                    }
                  }
                  type="text"
                  value=""
                />
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
  </div>
</div>
`;
